app/main.py

```diff
@@ -1,13 +1,40 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {("line" if error_name == "line_number"
+            else "column" if error_name == "column_number"
+            else "message" if error_name == "text"
+            else "name" if error_name == "code"
+            else "source" if error_name == "physical_line"
+            else None): ("flake8" if error_name == "physical_line"
+            else error_body)
+            for (error_name, error_body) in error.items()
+            if error_name != "filename"}
```

Comment:
You don't need to iterate through all error items
You can just make:
`{
     "line": error["line_number"]
 }
` etc.